lib/driver.rb

```diff
@@ -43,5 +45,31 @@ def add_trip(trip)
 
       @trips << trip
     end
+
+    def set_status_unavailable
```

Comment:
This method is called `set_status_unavailable`, but it only returns `true` or `false`.